app/views/layouts/user_mailer.html.slim

```diff
@@ -94,9 +94,11 @@ html xmlns="http://www.w3.org/1999/xhtml"
                                       tr
                                         th
                                           p.text-center
-                                            a href="#" = t('mailer.about', app: APP_NAME)
+                                            a href="#{MarketingSite.base_url}"
```

Comment:
That's what I went for at first but couldn't get it to work for some reason. The email wouldn't even get sent! If you know how, please let me know. I'll try it again too.